docs/getting-started/accessing-resources/getting-access-to-data.md

```diff
@@ -103,20 +141,145 @@ To list all available releases, you can use the following command from the root
+
+To download results for one or more modules, use the `--modules` option as follows:
+
+```sh
+# Get results from a single module
+./download-results.py --modules module-name
+
+# Get results from several modules
+./download-results.py --modules first-module,second-module
+```
+
+To list all available modules for which you can download results, you can use the `--list-modules` option as follows.
+This will list all modules available in, by default, the current release:
+
+```sh
+./download-results.py --list-modules
+```
+
+To see the result files that would be downloaded without performing the download yet, you can use the `--dryrun` option as follows:
+
+```sh
+./download-results.py --modules module-name --dryrun
+```
+
+While the structure of results files varies by module, some modules' results will be organized by project and/or sample.
+For those modules, you can use either the `--projects` or `--samples` flag (noting that these options are mutually exclusive) to download results specific to one or more projects or samples:
+
+```sh
+# use the --projects option
+./download-results.py --modules module-name --dryrun --projects SCPCPXXXXXX
+
+# use the --samples option
+./download-results.py --modules module-name --dryrun --samples SCPCSXXXXXX,SCPCSXXXXXY
+```
+
+
+#### Download result data structure
+
+Results will download into the `data` directory in your local copy of the `OpenScPCA-analysis` repository.
+Result downloads will generally follow this structure:
+
+```sh
+data
+├── {Release}
+│       └── results
+│          └── {Module name}
+│              └── {Result files}
+└── current -> {Release}
+```
+
+Both result files and ScPCA data files follow the same release schedule.
+Therefore, if you have downloaded both data and workflow results, all files will be present in the `data` directory with this overall structure:
+
+```sh
+data
+├── {Release}
+│       ├── {Project ID}
+│       │   └── {Sample ID}
+│       │   │     └── {Library files}
+│       │   ├── bulk_metadata.tsv (if applicable)
+│       │   ├── bulk_quant.tsv (if applicable)
+│       │   └── single_cell_metadata.tsv
+|       └── results
+|           └── {Module name}
+│              └── {Result files}
+└── current -> {Release}
+```
+
+To list all available releases, you can use the following command from the root of the repository:
+
+```sh
+./download-results.py --list-releases
+```
+
+The `data/current` directory is symlinked to the current release directory
+.
+This is generally the path you should use in your code.
+
+
+## Accessing test data
+
+The test data are simulated or permuted data files with the same structure as the real project data, and are generally much smaller than the original data (learn more at the [`simulate-sce`](https://github.com/AlexsLemonade/OpenScPCA-analysis/tree/main/analyses/simulate-sce) module).
```

Comment:
I think worth adding a sentence to say that these data are used during automated testing of modules.

Reply:
https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/503/commits/eedad84e8994f9ba66af6c236fadeed9f815d248